client/src/pages/components/SearchEnergizers.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+const SearchEnergizers = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [energizers, setSearchEnergizers] = useState([]);
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+
+
+
+  const filteredEnergizers = energizers.filter((energizer) => {
```

Comment:
Would this not only run once when the component is first instantiated, i.e. it should it not run when you do the handleSearch (on change) above and update the state? 

Also don't you need to maintain both a state of all energizers and the filtered ones?